examples/event-migration/build.sbt

```diff
@@ -1,6 +1,13 @@
+import Dependencies._
+import sbt.VirtualAxis.ScalaVersionAxis
+
+lazy val supportedScalaVersions = List(scalaVersion213, scalaVersion212)
+lazy val testScalaVersions = supportedScalaVersions ++ List("2.12.13", "2.12.14", "2.13.2", "2.13.3", "2.13.4", "2.13.5", "2.13.6", "2.13.7")
```

Comment:
Hmm do we also test `examples/` against multiple Scala versions? AFAICS it's just 
```yaml
      - name: Run examples
        run: sbt test
        working-directory: examples/event-migration
```

Reply:
you're right - I just reverted changes on this file